src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -42,6 +42,9 @@
     <Compile Include="System\Runtime\Serialization\SerializationException.cs" />
     <Compile Include="System\Runtime\Serialization\StreamingContext.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='' Or '$(TargetGroup)'=='netstandard1.7'">
```

Comment:
yeah, basically for now we should never have the second case, I suppose he did it like this so that it continues to work in the future when the default is not netstandard1.7 anymore, but I agree with Wes that for now we should keep it as only checking if it is empty